egeria-connector-jdbc/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/jdbc/ffdc/JdbcConnectorAuditCode.java

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.adapters.connectors.integration.jdbc.ffdc;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.AuditLogMessageDefinition;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.AuditLogMessageSet;
+import org.odpi.openmetadata.repositoryservices.auditlog.OMRSAuditLogRecordSeverity;
+
+
+/**
+ * The JdbcConnectorAuditCode is used to define the message content for the Audit Log.
+ *
+ * The 5 fields in the enum are:
+ * <ul>
+ *     <li>Log Message Id - to uniquely identify the message</li>
+ *     <li>Severity - is this an event, decision, action, error or exception</li>
+ *     <li>Log Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the situation</li>
+ *     <li>UserAction - describes how a user should correct the situation</li>
+ * </ul>
+ */
+public enum JdbcConnectorAuditCode implements AuditLogMessageSet {
+
```

Comment:
I think a lot of these messages are very code-centric. I think as we develop & get more experience with the connector we can later improve/refactor to have better end-user recommendations & clarity of overall impact. However as this is a first step I think it's fine to get started...

Reply:
#180